### DISCUSSION TITLE
Waiting for approval of Students Developer pack

### DISCUSSION THREAD

**Discussion by Bonged** [community/community#138958] [GitHub Education]
### Hi everyone,

Hi, I didn’t receive any communication after the approval of the student developer pack, Please speed up the process

**Comment by samueltexa** [comment]
Hi @Bonged,
The approval process for the GitHub Student Developer Pack can sometimes take a bit of time. My application took around two weeks to be processed, so it’s not uncommon for there to be a delay. However, rest assured it will approved provided the credentials are valid.

**Comment by Bonged** [comment]
Thanks for your reply


On Wed, 18 Sep 2024 at 10:07 AM, Alinda Samuel ***@***.***>
wrote:

> Hi @Bonged <https://github.com/Bonged>,
> The approval process for the GitHub Student Developer Pack can sometimes
> take a bit of time. My application took around two weeks to be processed,
> so it’s not uncommon for there to be a delay. However, rest assured it will
> approved provided the credentials are valid.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/138958#discussioncomment-10677169>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BLJX3T7EQGQS2YWXIIFX573ZXD7QDAVCNFSM6AAAAABOMZA4Q2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANRXG4YTMOI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

-- 
**This email is governed by the Disclaimer Terms of  SIU which may be 
viewed at **http://www.siu.edu.in/downloads/email-disclaimer.php 
<http://www.siu.edu.in/downloads/email-disclaimer.php>
